src/ec/packages/ec-component-text-area/adapter.js

```diff
@@ -13,10 +13,14 @@ const adapter = initialData => {
     delete adaptedData.is_disabled;
   }
 
-  adaptedData.helper_text = "This is the input's helper text";
+  adaptedData.helper_text = "This is the input's helper text.";
   adaptedData.optional_text = '(optional)';
-  adaptedData.name = 'example-name';
   adaptedData.label = 'Label';
+  adaptedData.required = true;
+  adaptedData.required_text = '*';
+  adaptedData.optional_text = '(optional)';
+  adaptedData.id = 'example';
```

Comment:
Why if we set it here we don't use this in the story, then..? (i know it's weird that we have ids in the specs but then we need to override them to make the diff match).

Reply:
Keeping this id for the story also fine.I just overridden the story because i thought it the story itself need to match the previous state.